api/db/migrate/20181005100009_add_elo_rating_to_player.rb

```diff
@@ -0,0 +1,5 @@
+class AddEloRatingToPlayer < ActiveRecord::Migration[5.2]
+  def change
+    add_column :players, :elo_rating, :integer, null: false, default: 1200
```

Comment:
I'd consider adding separate defender and attacker rating.

Reply:
@mhib could you explain more?